src/toil/cwl/cwltoil.py

```diff
@@ -1033,7 +1043,9 @@ def __str__(self) -> str:
 
 class ToilCommandLineTool(ToilTool, cwltool.command_line_tool.CommandLineTool):
     """Subclass the cwltool command line tool to provide the custom ToilPathMapper."""
-
+    def __init__(self, toolpath_object: 'CommentedMap', loadingContext: 'cwltool.context.LoadingContext'):
+        loadingContext.relax_path_checks = True
```

Comment:
This is an interesting policy decision you've made.

In cwltool this defaults to `False` unless the `--relax-path-checks` option is passed.

This is to encourage the use of unsurprising path components as other tools might have issues with spaces, emojis, colons, etc.. 

For running the CWL Conformance tests with `cwltool` we do of course [set `--relax-path-checks`](https://github.com/common-workflow-language/cwl-v1.2/blob/0d538a0dbc5518f3c6083ce4571926f65cb84f76/.github/workflows/cwltool.yml#L18)

Reply:
Yeah, I wasn't aware that `--relax-path-checks` is set for cwltool's conformance tests, I think that explains why toil-cwl-runner didn't pass the conformance tests, as I don't believe toil-cwl-runner currently applies that on CI. This can be removed then, and I can pass `--relax-path-checks` in Toil's CWL tests instead.